arelle/plugin/loadFromOIM.py

```diff
@@ -96,7 +96,7 @@
     "xbrl": nsOims,
     "xs": (XbrlConst.xsd,),
     "enum2": XbrlConst.enum2s,
-    "oimce": nsOimCes,
+    # "oimce": nsOimCes,
```

Comment:
Remove?

Reply:
This causes a couple of the conformance suite tests to fail.

Reply:
This causes a couple of the conformance suite tests to fail.